@navikt/core/react/src/toggle-group/parts/ToggleItem.tsx

```diff
@@ -45,7 +86,14 @@ const ToggleItem = forwardRef<HTMLButtonElement, ToggleGroupItemProps>(
           className="navds-toggle-group__button-inner"
           size={ctx?.size}
         >
-          {children}
+          {children ? (
+            children
+          ) : (
+            <>
+              {icon}
+              {label}
+            </>
+          )}
```

Comment:
Could be simplified to this, but only if we think it's equally readable:
```suggestion
          {children ?? (
            <>
              {icon}
              {label}
            </>
          )}
```